.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-  "dart.flutterSdkPath": "~/.asdf/installs/flutter/3.29.3-stable"
+  "dart.flutterSdkPath": "c:\\flutter"
```

Comment:
We should add this file in `.gitignore`file because windows and macos don't use the same path